mappings/net/minecraft/network/packet/s2c/play/NbtQueryS2CPacket.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/unmapped/C_ndeufatn net/minecraft/network/packet/s2c/play/NbtQueryResponseS2CPacket
+CLASS net/minecraft/unmapped/C_ndeufatn net/minecraft/network/packet/s2c/play/NbtQueryS2CPacket
```

Comment:
Is this a response to a C2S query packet?

Reply:
yes, my mistake